src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -414,7 +421,19 @@ private TypeWithAnnotations ComputeReturnType(DiagnosticBag diagnostics)
             else
             {
                 var binder = GetBinder();
-                return TypeWithAnnotations.Create(binder.GetSpecialType(SpecialType.System_Void, diagnostics, this.GetSyntax()));
+                var type = TypeWithAnnotations.Create(binder.GetSpecialType(SpecialType.System_Void, diagnostics, this.GetSyntax()));
+
+                if (IsInitOnly)
+                {
+                    var isInitOnlyType = Binder.GetWellKnownType(this.DeclaringCompilation,
```

Comment:
>GetWellKnownType [](start = 48, length = 16)

I don't think this function implements the rules stated in the speclet:
If there are multiple types by this name then the compiler will tie break in the following order:
1. The one defined in the project being compiled
2. The one defined in corelib

In particular, I do not think it gives priority to types in the corelib. I also think that it is better to follow the rules that we already use for other well-known types and suggest that we adjust the speclet instead.

Reply:
I know. There is already a PROTOTYPE comment in the test file tracking this. 
I agree that the standard rules are better. In the current design, there is no scenario for using new compiler on old TFM with your own IsExternalInit type, so I think the problem largely goes away.
I'll confirm with Jared.

---
In reply to: [415127279](https://github.com/dotnet/roslyn/pull/43275#discussion_r415127279) [](ancestors = 415127279)

Reply:
> I know. There is already a PROTOTYPE comment in the test file tracking this. 

I didn't see any PROTOTYPE comment specifically about this issue. 

---
In reply to: [415130972](https://github.com/dotnet/roslyn/pull/43275#discussion_r415130972) [](ancestors = 415130972,415127279)

Reply:
" // PROTOTYPE(init-only): test with ambiguous IsExternalInit types"

---
In reply to: [415215897](https://github.com/dotnet/roslyn/pull/43275#discussion_r415215897) [](ancestors = 415215897,415130972,415127279)